digitalpy/core/parsing/impl/hierarchical_format.py

```diff
@@ -1,6 +1,7 @@
-from digitalpy.parsing.impl.abstract_format import AbstractFormat
-from digitalpy.model.object_id import ObjectId
-from digitalpy.model.node import Node
+from digitalpy.core.parsing.abstract_format import AbstractFormat
+from digitalpy.core.domain.object_id import ObjectId
+from digitalpy.core.domain.node import Node
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position